packages/itwinui-react/src/core/Toast/Toaster.tsx

```diff
@@ -139,38 +139,41 @@ export default class Toaster {
       },
       ...(this.settings.order === 'descending' ? this.toasts : []),
     ];
+
+    // cannot `await` this, for backwards compat with the return value
     this.updateView();
+
     return { close: () => this.closeToast(currentId) };
```

Comment:
i wanted to make all methods `await`able, but that would cause breaking changes for users who are using the return value like this:

```jsx
const { close } = toaster.positive();
```

i thought about keeping backwards compatibility by adding a promise as a new property in the return value. but no one has asked for it and i don't see the benefit yet, so i decided against it.

```suggestion
    const updatePromise = this.updateView();
    const done = new Promise<boolean>(async (resolve) => {
      await updatePromise;
      resolve(true);
    });

    return { close: () => this.closeToast(currentId), done };
```